### PR TITLE
Populate recursively / Add sudo to setup script

### PR DESCRIPTION
**Pull request by OwenCalvin**
# My PR is a:

### 🐛 Bug fix
The setup script only work with sudo (on Windows WSL)

### 💅 Enhancement
Now the returned object is populated recursively  
_Example:  
You had an object with a relation to itself (tree system).  
When you get it, the returned object is auto populated with the childs, childs of child, ..._

### CODE REVIEW

File: scripts/setup.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const shell = require('shelljs');
-// Check npm version
-const npm = shell.exec('npm -v').stdout;
+// Check sudo npm version
+const npm = shell.exec('sudo npm -v').stdout;
 
 if (parseFloat(npm) < 5) {
-  throw new Error('[ERROR: Strapi] You need npm version @>=5');
```
Comment (derrickmehaffy) [inline scripts/setup.js]: Shouldn't add sudo into comments

File: scripts/setup.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 const shell = require('shelljs');
-// Check npm version
```
Comment (derrickmehaffy) [inline scripts/setup.js]: Shouldn't add sudo into comments

**Comment by Aurelsicoko** [comment]
How can we set the level of depth for the population?

**Comment by lauriejim** [comment]
@OwenCalvin Can you give an answer to @Aurelsicoko please.

**Pull request closed without merging**